docs/checkers/index.md

```diff
@@ -117,6 +117,24 @@ Corrected version:
 print_greeting(name="Arthur")
 ```
 
+(W0130)=
+
+### Duplicate Value (W0130)
```

Comment:
I recommend putting this immediately after the entry for "Duplicate key" (W0109)